__tests__/integration.test.js

```diff
@@ -404,4 +404,45 @@ describe('DELETE /api/users', () => {
             })
         })
     })
+})
+
+
+describe('GET /api/articles?topic=', () => {
+    test('GET200: endpoint response is 200 status with an array containing topics matching that query', () => {
+        const topic = 'mitch'
+
+        return request(app)
+        .get(`/api/articles?topic=${topic}`)
+        .expect(200)
+        .then(({ body }) => {
+            const { articles } = body
+
+            expect(articles.length).toBe(12)
+
+            articles.forEach(article => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: topic,
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                })
+            });
+        })
+    })
+
+    test('GET200: endpoint response is 200 empty array for a topic with no items', () => {
```

Comment:
this is actually a non existent article, but what you've written in your test description is absolutely another test we want, but for a valid topic with no articles (paper is on in the test db). This example though should be a 404